runtime-config-linux.md

```diff
@@ -146,45 +146,128 @@ The cgroups will be created if they don't exist.
 
 `cgroupsPath` can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
 
-Optionally, cgroups limits can be specified via `resources`.
+You can configure a container's cgroups via the `resources` field of the Linux configuration.
+Do not specify `resources` unless limits have to be updated.
+For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+
+#### Disable out-of-memory killer
 
 ```json
-    "resources": {
-        "disableOOMKiller": false,
-        "memory": {
-            "limit": 0,
-            "reservation": 0,
-            "swap": 0,
-            "kernel": 0,
-            "swappiness": -1
-        },
-        "cpu": {
-            "shares": 0,
-            "quota": 0,
-            "period": 0,
-            "realtimeRuntime": 0,
-            "realtimePeriod": 0,
-            "cpus": "",
-            "mems": ""
-        },
-        "blockIO": {
-            "blkioWeight": 0,
-            "blkioWeightDevice": "",
-            "blkioThrottleReadBpsDevice": "",
-            "blkioThrottleWriteBpsDevice": "",
-            "blkioThrottleReadIopsDevice": "",
-            "blkioThrottleWriteIopsDevice": ""
-        },
-        "hugepageLimits": null,
-        "network": {
-            "classId": "",
-            "priorities": null
-        }
-    }
+	"disableOOMKiller": false
 ```
 
-Do not specify `resources` unless limits have to be updated.
-For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+#### Memory
+
+```json
+	"memory": {
+		"limit": 0,
```

Comment:
I think our other examples use four-space indents for JSON, so it's probably better to stick with that and skip the tabs.